extensions/amazon-lambda-xray/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: "AWS Lambda Xray"
+metadata:
+  keywords:
+  - "lambda"
+  - "aws"
+  - "amazon"
+  - "xray"
+  categories:
+  - "cloud"
+  guide: "https://quarkus.io/guides/amazon-lambda-xray"
```

Comment:
I don't think you have a guide for that one? In this case, just remove the line.